components/TemplateSection/TemplateSection.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
```

Comment:
I think this is more readable
`import { ReactNode } from "react"`
Also, out of consistency with the imports of `useState`, etc